internal/source/git.go

```diff
@@ -36,9 +41,24 @@ func (r *Git) Unpack(ctx context.Context, bundle *rukpakv1alpha1.Bundle) (*Resul
 
 	// Set options for clone
 	cloneOpts := git.CloneOptions{
-		URL:      gitsource.Repository,
-		Progress: os.Stdout,
-		Tags:     git.NoTags,
+		URL:             gitsource.Repository,
+		Progress:        os.Stdout,
+		Tags:            git.NoTags,
+		InsecureSkipTLS: bundle.Spec.Source.Git.TLSNoVerify,
+	}
+
+	if bundle.Spec.Source.Git.SecretName != "" {
+		userName, accessToken, err := r.getCredential(ctx, bundle)
```

Comment:
Rather than `accessToken` terminology, can we use `password` everywhere (here, and as the key in the `secret.data` map). I think `password` is the generally accepted terminology that the git project uses in docs about authentication.

Some git hosts (e.g. github) support access tokens as passwords, but I think access token is a more specific term than password, and not one we should enshrine in the API.

Reply:
In fact, there's a specific secret type in kubernetes that enforces these fields: https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret

Let's align with that.